### PR TITLE
Fix compute_resource RHVProviderForm

### PR DESCRIPTION
**Pull request by ldjebran**
Close https://github.com/SatelliteQE/robottelo/issues/6178

tests output at https://github.com/SatelliteQE/robottelo/pull/6179

### CODE REVIEW

**Comment by oshtaier** [comment]
hmm, a little bit weird implementation. Let's me think if we can improve it

File: airgun/views/computeresource.py [hunk 1]
```diff
@@ -108,10 +108,30 @@ class RHVProviderForm(View):
         password = TextInput(id='compute_resource_password')
         api4 = Checkbox(id='compute_resource_use_v4')
         load_datacenters = Text("//*[contains(@id,'test_connection_button')]")
+        datacenter = FilteredDropdown(id='s2id_compute_resource_uuid')
         certification_authorities = TextInput(id='compute_resource_public_key')
 
+        # some of the widgets values need to be filled after the main values
```
Comment (oshtaier) [inline airgun/views/computeresource.py]:
```
url = TextInput(id='compute_resource_url')
user = TextInput(id='compute_resource_user')
password = TextInput(id='compute_resource_password')
api4 = Checkbox(id='compute_resource_use_v4')
certification_authorities = TextInput(id='compute_resource_public_key')
@View.nested
class datacenter(View):
    load_datacenters = Text("//a[contains(@id,'test_connection_button')]")
    datacenter_value = FilteredDropdown(id='s2id_compute_resource_uuid')

    def before_fill(self, values=None):
        self.load_datacenters.click()
```
then
`provider_content.datacenter.datacenter_value: 'value'`
 
All widgets are filled in strict order, one by one, so `datacenter` view will be filled last

**Comment by oshtaier** [comment]
That should work, maybe it will require small fixes, but I believe not
Anyway, we cannot allow proposed code in initial commit, sorry

File: airgun/views/computeresource.py (hunk 1 above)
Comment (ldjebran) [reply airgun/views/computeresource.py]: @oshtaier thanks, this is a more simple implementation

**Comment by ldjebran** [comment]
@oshtaier made the changes but the tests are blocked (we cannot resolve the compute resource host name) https://github.com/SatelliteQE/robottelo/pull/6179/comment#issuecomment-411708609

File: airgun/views/computeresource.py [hunk 2]
```diff
@@ -107,11 +107,16 @@ class RHVProviderForm(View):
...
         api4 = Checkbox(id='compute_resource_use_v4')
-        load_datacenters = Text("//*[contains(@id,'test_connection_button')]")
         certification_authorities = TextInput(id='compute_resource_public_key')
 
-        def after_fill(self, was_change):
-            self.load_datacenters.click()
+        @View.nested
+        class datacenter(View):
+            load_datacenters = Text(
+                "//*[contains(@id,'test_connection_button')]")
```
Comment (oshtaier) [inline airgun/views/computeresource.py]: as I mentioned in my comment - `//a[contains(@id,'test_connection_button')]`

**Comment by ldjebran** [comment]
changes done : test run out put https://github.com/SatelliteQE/robottelo/pull/6179#issuecomment-411759293

**Pull request merged**